reference/carts.v3.yml

```diff
@@ -881,71 +846,21 @@ components:
               type: number
             product_id:
               type: number
-            variant_id:
-              type: number
-              description: Required if the product has variants
             list_price:
               type: number
             option_selections:
               type: array
               items:
                 type: object
+                title: Product Option Selection
                 properties:
                   option_id:
                     type: number
                   option_value:
                     type: string
-                title: Product Option Selection
           required:
             - quantity
             - product_id
-            - variant_id
-        gift_certificate:
```

Comment:
checking in to make sure this section wasn't removed by accident, since I don't think I see it elsewhere.